components/Wallet/Send.js/ConfirmationCard.jsx

```diff
@@ -90,36 +79,29 @@ const ConfirmationCard = ({ value, walletType, toAddress }) => {
             Confirmation
           </Text>
         </Box>
-        <Stepper
-          textColor='card.confirmation.foreground'
-          completedDotColor='card.confirmation.foreground'
-          incompletedDotColor='core.silver'
-          step={2}
-          totalSteps={2}
-        >
-          Step 2
-        </Stepper>
+        <Box display='flex' alignItems='center'>
+          <Stepper
+            textColor='card.confirmation.foreground'
+            completedDotColor='card.confirmation.foreground'
+            incompletedDotColor='core.silver'
+            step={2}
+            totalSteps={2}
+          />
+          <Box width={5} mx={2} />
```

Comment:
What's this for?

Reply:
This matches the width of the `CloseButton` component in the SendCard so that the Stepper components across the Send and Confirmation cards align vertically